app/src/ui/open-pull-request/open-pull-request-dialog.tsx

```diff
@@ -208,6 +208,7 @@ export class OpenPullRequestDialog extends React.Component<IOpenPullRequestDialo
         className="open-pull-request"
         onSubmit={this.onCreatePullRequest}
         onDismissed={this.props.onDismissed}
+        maxWidth={2000}
```

Comment:
Arbitrary choice based on something that looked ok my monitor. :D Any feedback regarding a way to determine a good max width? Should we just go with infinity?

Reply:
Chose infinity for PR modal - since with other diffing we stretch for whole app.